src/layout/layout.scss

```diff
@@ -0,0 +1,208 @@
+// Layout component
+
+.Layout {
+  display: grid;
+  // stylelint-disable-next-line primer/no-undefined-vars
+  --Layout-sidebar-width: #{map-get($sidebar-width, "sm")};
+  // stylelint-disable-next-line primer/no-undefined-vars
+  --Layout-gutter: 16px;
```

Comment:
I'm ignoring this linter for now, would love a direction on how to properly add those variables